src/planwise/client/projects/views.cljs

```diff
@@ -16,6 +20,29 @@
             [planwise.client.projects.components.sidebar
              :refer [sidebar-section]]))
 
+(defn geojson-bbox-callback [isochrones filters threshold]
```

Comment:
This makes an API call in the `views` namespace which seems a bit out of place, but it's true that there's no other clear place to put it in. Maybe as a subscription?